File: src/main/java/com/jozufozu/exnihiloomnia/client/renderers/TileEntityBarrelRenderer.java
package com.jozufozu.exnihiloomnia.client.renderers;

import com.jozufozu.exnihiloomnia.common.blocks.barrel.logic.TileEntityBarrel;
import net.minecraft.client.renderer.tileentity.TileEntitySpecialRenderer;

public class TileEntityBarrelRenderer extends TileEntitySpecialRenderer<TileEntityBarrel>
{
    @Override
    public void render(TileEntityBarrel te, double x, double y, double z, float partialTicks, int destroyStage, float alpha)
    {
        te.getState().draw(te, x, y, z, partialTicks);
        
        super.render(te, x, y, z, partialTicks, destroyStage, alpha);
    }
}

File: src/main/kotlin/com/jozufozu/exnihiloomnia/client/tileentities/TileEntityBarrelRenderer.kt
package com.jozufozu.exnihiloomnia.client.tileentities

import com.jozufozu.exnihiloomnia.common.blocks.barrel.BarrelTileEntity
import net.minecraft.client.renderer.tileentity.TileEntitySpecialRenderer

class TileEntityBarrelRenderer : TileEntitySpecialRenderer<BarrelTileEntity>() {
    override fun render(te: BarrelTileEntity, x: Double, y: Double, z: Double, partialTicks: Float, destroyStage: Int, alpha: Float) {
        te.state.draw(te, x, y, z, partialTicks)

        super.render(te, x, y, z, partialTicks, destroyStage, alpha)
    }
}
